Sources/Rectangle.swift

```diff
@@ -19,7 +20,16 @@ public class Rectangle : Shape {
         width = width * scaleFactor
     }
     
-    public func getArea() -> Int {
-        return height * width
+    public func getArea() -> Float {
+        return Float(height * width)
+    }
+    
+    public func getVolume() throws -> Float {
+        throw SpaceTimeError()
```

Comment:
Is this a real error?

Reply:
A better error might be the aspect ratio error. Spacetime seems incorrect to me.

Reply:
We should either finish getVolume to actually return the volume or remove entirely, currently only an error is thrown.

Reply:
Get rid of volume, 2d objects have no volume.